install/sensors.csv

```diff
@@ -818,7 +818,9 @@ Licor,Li-850 CO2/H2O Gas Analyzer,HGA-06919,TO006,05,0,,0,0,0,0,0,0,2024-06-04T0
 Licor,Li-850 CO2/H2O Gas Analyzer,HGA-06920,TO003,12,0,,0,0,0,0,0,0,2024-03-01T00:20:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan TACCL-BH1,2363,CBGS,22,0,,0,21.8,0,0,0,0,2023-02-23T03:16:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan TACCL-N1,2590,CBGS,21,0,,0,0,0,0,0,0,2023-02-23T03:16:00Z,9999-01-01T00:00:00Z
+Nanometrics,Titan TACCL-N1,3397,CTPS,20,0,,0,0,0,0,0,0,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan TACCL-N1,3399,SMHS,20,230,,0,0,0,0,0,0,2024-07-01T03:10:00Z,9999-01-01T00:00:00Z
+Nanometrics,Titan TACCL-PH2,3526,CTPS,21,33,,0,0,0,0,0,0,2024-09-23T00:00:00Z,9999-01-01T00:00:00Z
```

Comment:
just to double check if azimuth is `33` (it is not indicated in the ticket --or perhaps I missed that detail)